### ISSUE TITLE
Option to skip dummy ASL scans

### ISSUE THREAD

**Issue by jan-petr** [ExploreASL/ExploreASL#354]
Create an option for skipping ASL scans.
These can be used to skip the dummy and noise scans Wibeke has.

### Release summary
Added an option x.DummyPositionInASL4D that removes marked dummy scans when splitting ASL to ASL+M0+dummy

**Comment by jan-petr** [comment]
I will use the DataPar parameter
DummyScanPositionInASL4D

And simply remove it in the same function where
M0PositionInASL4D
is removed (i.e. move M0 to M0, backup ASL in ASL_ORI and leave out the dummy scans).

**Comment by MichaelStritt** [comment]
@jan-petr: can you add this to the DataParTemplate as soon as it is implemented? We can also add it within #355.

**Comment by jan-petr** [comment]
> @jan-petr: can you add this to the DataParTemplate as soon as it is implemented? We can also add it within #355.

Yes, I will add it - all new parameters go the the DataParTemplate. But I'll made that edit within this issue to avoid confusion.

**Comment by jan-petr** [comment]
I made a proper parameter admin. JSON can have numeric arrays inputted both as a string or as a real numeric array. The condition to have it as string was thus not making any sense.

I now set both the M0 and Dummy Position by default to zero and deal with parameter checking inside the split function xASL_io_SplitASL_M0.

DummyScanPosition is added as a second parameter to this function - because these two positions are both absolute in the original ASL, so they need to be restructured both at the same time.

I have fixed several minor things and headers as well inside that function.
